master/getting-started/kubernetes/installation/manifests/app-layer-policy/istio-inject-configmap.yaml

```diff
@@ -0,0 +1,126 @@
+        resources: {}
+        securityContext:
+          privileged: true
+      containers:
+      - name: istio-proxy
+        image: quay.io/calico/istio-proxy_debug:20180118230747
+        args:
+        - proxy
+        - sidecar
+        - --configPath
+        - {% raw %}{{ .ProxyConfig.ConfigPath }}
+        - --binaryPath
+        - {{ .ProxyConfig.BinaryPath }}
+        - --serviceCluster
+        {{ if ne "" (index .ObjectMeta.Labels "app") -}}
+        - {{ index .ObjectMeta.Labels "app" }}
+        {{ else -}}
+        - "istio-proxy"
+        {{ end -}}
+        - --drainDuration
+        - {{ formatDuration .ProxyConfig.DrainDuration }}
+        - --parentShutdownDuration
+        - {{ formatDuration .ProxyConfig.ParentShutdownDuration }}
+        - --discoveryAddress
+        - {{ .ProxyConfig.DiscoveryAddress }}
+        - --discoveryRefreshDelay
+        - {{ formatDuration .ProxyConfig.DiscoveryRefreshDelay }}
+        - --zipkinAddress
+        - {{ .ProxyConfig.ZipkinAddress }}
+        - --connectTimeout
+        - {{ formatDuration .ProxyConfig.ConnectTimeout }}
+        - --statsdUdpAddress
+        - {{ .ProxyConfig.StatsdUdpAddress }}
+        - --proxyAdminPort
+        - {{ .ProxyConfig.ProxyAdminPort }}
+        - --controlPlaneAuthPolicy
+        - {{ .ProxyConfig.ControlPlaneAuthPolicy }}{% endraw %}      
+        env:
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: INSTANCE_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.podIP
+        - name: USE_EDS_V1
+          value: 1
+        imagePullPolicy: IfNotPresent
+        securityContext:
+            privileged: true
+            readOnlyRootFilesystem: false
+            runAsUser: 1337
+        restartPolicy: Always
+        volumeMounts:
+        - mountPath: /etc/istio/proxy
+          name: istio-envoy
+        - mountPath: /etc/certs/
+          name: istio-certs
+          readOnly: true
+        - mountPath: /var/run/dikastes
+          name: dikastes-sock
+      - name: dikastes
+        image: {{site.imageNames["dikastes"]}}:{{site.data.versions[page.version].first.components["calico/dikastes"].version}}
+        args: ["/dikastes", "server", "-l", "/var/run/dikastes/dikastes.sock", "-d", "/var/run/felix/nodeagent/socket", "--debug"]
+        volumeMounts:
+        - mountPath: /var/run/dikastes
+          name: dikastes-sock
+        - mountPath: /var/run/felix
+          name: felix-sync
+      volumes:
+      - emptyDir:
+          medium: Memory
+        name: istio-envoy
+      - name: istio-certs
+        secret:
+          optional: true
+          {% raw %}{{ if eq .Spec.ServiceAccountName "" }}secretName: istio.default{{ else }}secretName: {{ printf "istio.%s" .Spec.ServiceAccountName }}{{ end }}{% endraw %}
```

Comment:
I don't think this `raw` is needed at all. It's currently rendering like this, which doesn't seem right:
```
        secret:
          optional: true
          {{ if eq .Spec.ServiceAccountName "" }}secretName: istio.default{{ else }}secretName: {{ printf "istio.%s" .Spec.ServiceAccountName }}{{ end }}
```

Reply:
If this is supported my mind is going to explode

Reply:
No, `kubectl` doesn't but the pod that consumes this configmap is itself a kind of templating engine. That's why this configmap has template tags in it that need to be escaped.

Reply:
ohh, i didn't notice this was just technically a string inside a configmap. phew. Though i kind of wish it did work that way :smile: 

So we can dismiss the original comment, it's fine as is.